packages/tooltip/src/Tooltip.tsx

```diff
@@ -67,7 +66,7 @@ const notchVariants = {
 
 interface PopoverFunctionParameters {
   alignment: Align;
-  justification: Justification | Justify;
+  justification: Justify;
```

Comment:
Can we change this key to justify too since it's going to be a breaking change anyways?

Reply:
should we also change `alignment` to `align` then?

Reply:
I'm in favor of that. I think our internal names are good enough to be the
same that's used externally, right?

Reply:
Actually, nice catch! It looks like I'd already changed it in `Popover` to be `justify` and TS somehow didn't catch it...